api/docs.go

```diff
@@ -3813,6 +3869,10 @@ const docTemplate = `{
                     "description": "Timestamp of last introspection that had updates",
                     "type": "string"
                 },
+                "latest_snapshot_url": {
+                    "description": "Latest url for the Snapshot distribution",
```

Comment:
Most of the file has upper-case URL, and I think snapshot can be lower case as its just a term in common use.